.github/workflows/native-build-development.yml

```diff
@@ -90,11 +108,16 @@ jobs:
         shell: bash
         run: rm -rf ~/.m2/repository/org/acme
 
+  report:
+    name: Report
+    runs-on: ubuntu-latest
+    needs: [build-with-native]
+    if: "always() && github.repository == 'quarkusio/quarkus-quickstarts'"
+    steps:
       - name: Report
-        if: "always() && github.repository == 'quarkusio/quarkus-quickstarts'"
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_API_TOKEN }}
-          STATUS: ${{ job.status }}
+          STATUS: ${{ needs.build-with-native.result }}
         run: |
           echo "The report step got status: ${STATUS}"
           sudo apt-get update -o Dir::Etc::sourcelist="sources.list" \
```

Comment:
Could be switched to the less verbose way: https://github.com/quarkusio/quarkus/blob/main/.github/workflows/jdk-early-access-build.yml#L112-L119
But that's mostly offtopic for this PR.

Reply:
this can be done in another PR.